src/v2/view-builder/internals/BaseForm.js

```diff
@@ -100,6 +100,7 @@ export default Form.extend({
         FormUtil.applyShowWhen(this.last(), input.showWhen);
       }
     } else {
+      input["multirowError"] = true;
```

Comment:
Probably better to go here: https://github.com/okta/okta-signin-widget/blob/master/src/v2/ion/uiSchemaTransformer.js#L62